lib/osf-components/addon/components/file-actions-menu/template.hbs

```diff
@@ -6,7 +6,7 @@
     as |dropdown|
 >
     <dropdown.trigger
-        aria-label={{t 'file_actions_menu.actions' filename=@item.displayName}} 
```

Comment:
We sometimes have trailing spaces after a parameter/attribute in the template when the tag itself takes up multiple lines. The trailing space itself won't change how it renders. If you encounter it when you're working on some component then feel free to remove it. However if this is done by some VSC extension, or it is on some other components that you're not actively working on, I don't think we should include it in the PR because it would clutter the commit history and the diff. Again, very minor.

Reply:
That sounds good, I'm returning it to its original state now.